src/colorizer/ColorizeCanvas3D.ts

```diff
@@ -289,7 +289,7 @@ export class ColorizeCanvas3D implements IRenderCanvas {
       return;
     }
     const id = this.params.track ? this.params.track.getIdAtTime(this.currentFrame) : -1;
-    this.view3d.setSelectedID(this.volume, this.params.dataset.segmentationChannel ?? 0, id + 1);
```

Comment:
This was a bug caused by highlighted ids being off by one in the vole-core shader.